docs/docs/80-downloads.md

```diff
@@ -7,9 +7,10 @@
 ## Docker images
 
 Image variants:
-* The `next` images are based on the current master branch
-* The `latest` is same as next (will change after v0.15.0 release to be latest stable release)
+* The `latest` image is the latest stable release
 * The `x.x.x` version images are stable releases
+* The `x.x` images are based on version specific release branche, they have the latest bugfixes backported
```

Comment:
@anbraten what do you thing about the wording of this?

Reply:
I am not really a fan of tagging `x.x`. We currently do not tag `x.x` in git anyways and we would have to re-tag for `x.x` each time we want to have a new version if we want to do it with the current way.

Reply:
https://hub.docker.com/layers/woodpeckerci/woodpecker-server/v0.14/images/sha256-4f728b67930fefa034781d34fb8dcded53e83449e0d3944308f16c8ffdf62fd6?context=explore

and it woks similar to `next` - so no we dont have to re-tag it just publish on merge/push

Reply:
But which job will create it 🤔 
`publish-server` => build next on default branch
`release-server` =>  `v{tag-name}` created on tags

Reply:
I changed the  `next` target on the release branch:
https://github.com/woodpecker-ci/woodpecker/blob/release/v0.14/.woodpecker/main.yml#L62-L70

we have two options:
 - change it for each release branch specific to it
 - add new step for this whit generic filters (branch: `release/*` ...)

Reply:
Okay, then we should do the second option, so we do not accidentally forget it 
> add new step for this whit generic filters (branch: release/* ...)

Reply:
OK, I'll add a task to  #684